packages/election/graphql/resolvers/_mutations/submitCandidacy.js

```diff
@@ -0,0 +1,42 @@
+const getElections = require('../../../lib/getElections')
+const resolveCandidate = require('../../../lib/resolveCandidate')
+const { upsert } = require('../../../lib/db')
+
+const {
+  Roles,
+  ensureSignedIn
+} = require('@orbiting/backend-modules-auth')
+
+module.exports = async (_, { slug }, { pgdb, user: me, req }) => {
+  ensureSignedIn(req)
+  Roles.ensureUserIsInRoles(me, ['admin', 'associate'])
+
+  const election = (await getElections(pgdb, me, {slug}))[0]
```

Comment:
This is the 2nd time while reviewing this code in which we need only a single election. Would totally consider provide an util function for that.